pulp_rpm/tests/functional/api/test_domains.py

```diff
@@ -0,0 +1,288 @@
+import json
+
+import pytest
+import uuid
+
+from django.conf import settings
+
+from pulpcore.client.pulp_rpm import RpmRepositorySyncURL, ApiException
+from pulpcore.client.pulpcore.exceptions import ApiException as CoreApiException
+
+from pulp_smash.pulp3.utils import gen_repo
```

Comment:
```suggestion
```
I would avoid adding pulp_smash imports to new tests. `gen_repo` only provides a dict with `name=str(uuid.uuid4())`, so no need to import it.